data-prepper-plugins/s3-sink/src/main/java/org/opensearch/dataprepper/plugins/sink/S3SinkConfig.java

```diff
@@ -0,0 +1,101 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package org.opensearch.dataprepper.plugins.sink;
+import org.opensearch.dataprepper.model.configuration.PluginModel;
+import org.opensearch.dataprepper.plugins.sink.configuration.AwsAuthenticationOptions;
+import org.opensearch.dataprepper.plugins.sink.configuration.ObjectOptions;
+import org.opensearch.dataprepper.plugins.sink.configuration.ThresholdOptions;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+import jakarta.validation.Valid;
+import jakarta.validation.constraints.NotNull;
+
+/*
+    An implementation class of s3 sink configuration
+ */
+public class S3SinkConfig {
+
+    static final String DEFAULT_BUCKET_NAME = "dataprepper";

```

Comment:
The S3 bucket should not have a default. Only one AWS user in the entire world can have a bucket named `dataprepper`.

Reply:
Thanks, David for your review. Addressed.